src/rgw/rgw-restore-bucket-index

```diff
@@ -100,6 +100,7 @@ where:
   -b <bucketname>     Required - name of the bucket to operate on
   -l <rados-ls-file>  Optional - file containing the output of 'rados ls -p <pool>'
   -p <pool>           Optional - data pool; if not provided will be inferred from bucket and zone information
+  -r <realm-name>     Optional - specify the realm if not applying to the default realm"
```

Comment:
you'd have similar issues specifying a non-default `--rgw-zonegroup` or `--rgw-zone`

Reply:
Added. Thanks!